src/openhab.cpp

```diff
@@ -224,63 +211,63 @@ void OpenHAB::connect() {
             _flagOpenHabConnected = false;
             disconnect();
             qCDebug(m_logCategory) << "openhab not reachable";
+        } else if (reply->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt() == 200) {
+            qCDebug(m_logCategory) << reply;
+            qCDebug(m_logCategory) << reply->header(QNetworkRequest::ContentTypeHeader).toString();
+            qCDebug(m_logCategory) << reply->header(QNetworkRequest::LastModifiedHeader).toDateTime().toString();;
+            qCDebug(m_logCategory) << reply->header(QNetworkRequest::ContentLengthHeader).toULongLong();
+            qCDebug(m_logCategory) << reply->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt();
+            qCDebug(m_logCategory) << reply->attribute(QNetworkRequest::HttpReasonPhraseAttribute).toString();
```

Comment:
Please merge into a single qCDebug statement (<< can be chained).

Reply:
removed the unnecessary information